src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheck.java

```diff
@@ -56,7 +56,82 @@
  * <pre>
  * &lt;module name=&quot;IllegalCatch&quot;/&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * try {
+ *     // some code here
+ * } catch (Exception e) { // violation
+ *
+ * }
+ *
+ * try {
+ *     // some code here
+ * } catch (ArithmeticException e) {
+ *
+ * } catch (Exception e) { // violation, catching Exception is illegal
+ *                           and order of catch blocks doesn't matter
+ *
+ * }
+ *
+ * try {
+ *     // some code here
+ * } catch (ArithmeticException | Exception e) { // violation, catching Exception is illegal
  *
+ * }
+ *
+ * try {
+ *     // some code here
+ * } catch (ArithmeticException e) { // OK
+ *
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to extend default list with ArithmeticException:
```

Comment:
Sorry for confusion, this property is not for default list extension, but to override it.

1. Please change description to clearly show it.
2. Please change config example to non-default things (change Exception to something like OutOfMemoryError)

Reply:
@strkkk done, please review.